### ISSUE TITLE
Where is the session first created

### ISSUE THREAD

**Issue by bwyss** [angular-fullstack/generator-angular-fullstack#2444]
I am trying to understand the session workflow.

When I navigate to http://localhost/login, a session is created in mongodb. Where is the code that triggers the session creation?

**Comment by Awk34** [comment]
`express-session` makes the sessions in Mongo. Here's the code: https://github.com/angular-fullstack/generator-angular-fullstack/blob/4.1.1/templates/app/server/config/express.js#L60-L69

**Comment by bwyss** [comment]
Thanks for the quick reply, love the generator by the way.
That looks like the session configuration, It's not clear to me where / when sessions are being created?

**Comment by Awk34** [comment]
Right, the session *creation* is taken care of by [passport](https://github.com/jaredhanson/passport)
